### ISSUE TITLE
Debian upgrade instructions

### ISSUE THREAD

**Issue by faxm0dem** [openzfs/openzfs-docs#473]
If I'm not mistaken, there is no information on how to upgrade Debian from one major release to another.
If there's nothing special to do, it should at least be mentioned, what do you think?

**Comment by gmelikov** [comment]
Yes, during last 8 years I didn't need to do anything on dist-upgrades. Somebody may add such note to our docs, I'd be glad to review it.

**Comment by Markohs** [comment]
Yes that's exactly what I sent the patch request for, a previous install of the headers as apt doesn't warrantee the order of installation, so 1.6 is a new:

apt install --yes linux-headers-generic

and the old 1.6 is the new 1.7
